cli/popper/commands/cmd_run.py

```diff
@@ -15,7 +15,9 @@
 @click.command(
     'run', short_help='Run a workflow or action.')
 @click.argument(
-    'action', required=False)
+    'action_wfile',
```

Comment:
since this is shown in the help, let's name it `target` and then in the help description of the command we explain what this is